numpy/core/tests/test_regression.py

```diff
@@ -14,8 +14,8 @@
         assert_raises_regex, assert_warns, suppress_warnings,
         _assert_valid_refcount, HAS_REFCOUNT,
         )
+from numpy.testing._private.utils import _no_tracing
```

Comment:
Looks like the name has not been changed in the test.

Reply:
sorry, fixed